From 1ee9a437f947d0f63c27989db765a821f4c817ab Mon Sep 17 00:00:00 2001
From: "M. P. Halpin" <miles@Ms-MacBook-Pro.local>
Date: Sat, 31 Dec 2022 20:45:35 -0800
Subject: [PATCH 382/458] Stop beep when running iOS apps on ARM-based Macs

(cherry picked from commit bbf38bbbc334bc0ee90d3eec228a968d8fc1689e)
---
 src/video/uikit/SDL_uikitview.m | 4 ----
 1 file changed, 4 deletions(-)

diff --git a/src/video/uikit/SDL_uikitview.m b/src/video/uikit/SDL_uikitview.m
index c35864897..a862273d7 100644
--- a/src/video/uikit/SDL_uikitview.m
+++ b/src/video/uikit/SDL_uikitview.m
@@ -412,7 +412,6 @@ extern int SDL_AppleTVRemoteOpenedAsJoystick;
             SDL_SendKeyboardKey(SDL_PRESSED, scancode);
         }
     }
-    [super pressesBegan:presses withEvent:event];
 }
 
 - (void)pressesEnded:(NSSet<UIPress *> *)presses withEvent:(UIPressesEvent *)event
@@ -423,7 +422,6 @@ extern int SDL_AppleTVRemoteOpenedAsJoystick;
             SDL_SendKeyboardKey(SDL_RELEASED, scancode);
         }
     }
-    [super pressesEnded:presses withEvent:event];
 }
 
 - (void)pressesCancelled:(NSSet<UIPress *> *)presses withEvent:(UIPressesEvent *)event
@@ -434,13 +432,11 @@ extern int SDL_AppleTVRemoteOpenedAsJoystick;
             SDL_SendKeyboardKey(SDL_RELEASED, scancode);
         }
     }
-    [super pressesCancelled:presses withEvent:event];
 }
 
 - (void)pressesChanged:(NSSet<UIPress *> *)presses withEvent:(UIPressesEvent *)event
 {
     /* This is only called when the force of a press changes. */
-    [super pressesChanged:presses withEvent:event];
 }
 
 #endif /* TARGET_OS_TV || defined(__IPHONE_9_1) */
-- 
2.17.1

